invenio_rdm_records/resources/resources.py

```diff
@@ -149,7 +149,7 @@ def delete(self):
     @response_handler(many=True)
     def search(self):
```

Comment:
minor: I'm doubting if we should actually call this `read_all`... because in the end we are not passing any "search query"